### DISCUSSION TITLE
Allow Insecure Registry

### DISCUSSION THREAD

**Discussion by creib** [vitobotta/hetzner-k3s#202] [Q&A]
Hi,
in my case, I have a docker-registry running on the cluster. Therefore I used a Helm-chart to install it. Now I would like to deploy a new pod using an image from this private registry. I am able to achieve this by referencing the docker-registry using an ingress with certificate. It is working like this. Drawback of this solution is, that I have to allow the public ip address of the ClusterNode to access the ingress route.
As an alternative way, I am trying to pull the image using the service url like docker-registry.docker-registry.svc.cluster.local:5000. In this case pulling of the image fails, because it is a insecure registry. 

Is it possible to define in hetner-k3s allowing to pull from insecure registries?

Thanks for your help

**Comment by vitobotta** [comment]
Hi, what error/probem do you get if you try to pull images from that registry?
